Bottom: 1e36775cee5edd025eae793f0a997e660ad7adc7
Top:    be232ff429cca8897c4b106d1577f3ab35b1e23f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-18 20:44:17 +0000

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index 80cb6db..26b20cb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -386,8 +386,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
